local/languages/finna/en-gb.ini

```diff
@@ -908,6 +908,12 @@ R2_register_read_more_html = "<a href="%%url%%">Read more about the service</a>.
 R2_register_weak_loginmethod = "Finna Reading Room requires strong authentication. Log out and verify your identify using the Suomi.fi e-Identification service."
 R2_rems_connect_error = "An error occurred. Finna Reading Room component not responding. We apologise for any inconvenience caused."
 R2_rems_session_expired = "Your session was terminated for security reasons. You can log in to the Reading Room again."
+reserve_material = "Reserving material"
+reserve_material_arrival_info_html = "<div class="alert alert-info">The material will be delivered to the National Library’s Special Collections Reading Room. A National Library card is required for use (see the instructions). Read the rules governing use: <a href="https://www.kansalliskirjasto.fi/en/rules-special-collections-reading-room" target="_blank">https://www.kansalliskirjasto.fi/en/rules-special-collections-reading-room</a></div>"
+reserve_material_info = "This material can only be accessed in the National Library’s Special Collections Reading Room."
```

Comment:
Nämä tekstit eivät voi olla näin yleisissä käännöksissä. Joko vain placeholder paikallisesti laitettaville teksteille tai sitten esimerkkiteksti hyvin yleisellä tasolla.